UI/window-basic-main.cpp

```diff
@@ -2133,6 +2133,8 @@ OBSBasic::~OBSBasic()
 	delete multiviewProjectorMenu;
 	delete previewProjector;
 	delete studioProgramProjector;
+	if (scaleFilteringMenu)
```

Comment:
This check is redundant. You can `delete` a null pointer.

Reply:
thanks for pointing that out. I've removed all the unnecessary checks.